### DISCUSSION TITLE
plays/ammaaraslam/react-gradients/

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#494] [Play Feedback]
# plays/ammaaraslam/react-gradients/

React Gradients is list of beautiful gradients which you can easily use for your application by copying the CSS/TailwindCSS code for the desired gradient.

https://reactplay.io/plays/ammaaraslam/react-gradients/

**Comment by atapas** [comment]
Such a great project. Awesome 👏

**Reply by valeriyarass** [reply]
awesome!
